pkg/html/dynamic/document.go

```diff
@@ -556,11 +551,25 @@ func (doc *HTMLDocument) InputBySelector(selector values.String, value core.Valu
 		return values.False, err
 	}
 
-	if res.Type() == core.BooleanType {
-		return res.(values.Boolean), nil
+	if res.Type() == core.BooleanType && res.(values.Boolean) == values.False {
+		return values.False, nil
 	}
 
-	return values.False, nil
+	delayMs := time.Duration(delay)
+
+	time.Sleep(delayMs * time.Millisecond)
```

Comment:
Why do you need Sleep here?

Reply:
Oh, is it imitation of a pause before starting to type?

Reply:
Yes, simulating a delay between clicking and typing.